Cargo.lock

```diff
@@ -145,9 +145,12 @@ source = "registry+https://github.com/rust-lang/crates.io-index"
 checksum = "16b0a3d9ed01224b22057780a37bb8c5dbfe1be8ba48678e7bf57ec4b385411f"
 dependencies = [
  "iana-time-zone",
+ "js-sys",
  "num-integer",
  "num-traits",
  "serde",
+ "time",
+ "wasm-bindgen",
```

Comment:
Do we make use of these or have they been added because of running `cargo fetch` locally

Reply:
Yea I was trying to remove these manually but they keep getting added back when running `cargo check` or building with rust-analyzer. I think `time` is possibly getting pulled in by the use of `chrono` in the status controller's index.

Reply:
Ended up removing this with https://github.com/linkerd/linkerd2/pull/10236/commits/e0d242b5ba8e55de3787f6cc0599333b2cb82125.